parts/common/components.json

```diff
@@ -80,7 +80,7 @@
       "windowsVersions": [
         {
           "renovateTag": "registry=https://mcr.microsoft.com, name=oss/kubernetes/pause",
-          "latestVersion": "3.9"
+          "latestVersion": "3.9-hotfix-20230808"
```

Comment:
Does containerd config need to be updated with this?  Or it just grabs the latest 3.9 available?

Reply:
The pause image version is specified in an rp map, that (I assume) is passed to the CSE that creates the containerd config.